entries/a/axosinvest.com.json

```
@@ -1,12 +1,12 @@
 {
   "Axos Invest": {
     "domain": "axosinvest.com",
```

```suggestion
    "domain": "axosinvest.com",
    "additional-domains": [
      "axos.com"
    ],
```
Maybe remove the `axosinvest.com.svg` if you use `axosbank.com.svg`

---

The `axosinvest.com.svg` _has_ been removed. Otherwise a check would point out that there is an unused image.

---

Weirdly enough, `axosinvest.com` redirects to `axos.com/invest/`, but `axos.com` on its own redirects to `axosbank.com`. :thinking:

---

Probably and `axos.com` entry with `addition-domains` would resolve the issue.